vivisect/parsers/__init__.py

```diff
@@ -49,6 +49,9 @@ def sha256Bytes(bytes):
 )
 
 def guessFormat(bytes):
+    if bytes.startswith('MSGVIV'):
+        return 'mpviv'
```

Comment:
how's msgfile ever get loaded and used?  it's been a while since i looked extensively add how this works,but i think this should look for a module named vivisect.parsers.mpviv?

Reply:
So kind of per our discussion about keeping PRs more focused, I kept the PR focused on just the new module. I did note the fact that both basicfile and mpfile both do the whole "try and find vivisect.parsers.mpviv or vivisect.parsers.basicfile", andthen fail. But I decided to keep things focused and make that a different PR.